websites.yml

```diff
@@ -46,3 +46,40 @@ username_patterns:
     invalid_patterns:
       - "\\.(com|net)"
       - "(\\.)\\1{1,}"  # consecutive '.' not allowed
+avatar:
+  pinterest:
+    type: regex
```

Comment:
`type` isnt needed.  it can be inferred. we need `property:` , `url:` , otherwise the entries can be short and clear `gitlab: opengraph` , as that is the most common.